CHANGELOG.mkd

```diff
@@ -4,6 +4,8 @@ CHANGELOG
 Unreleased
 ----------
 
+- Make SVN source ignore unused options. [#1305](https://github.com/puppetlabs/r10k/pull/1302)
```

Comment:
Should this reference the issue instead of the PR?

Reply:
Yes that was what I meant 🤦‍♀️